frontend/src/app/application-forms/fields/quantity.component.html

```diff
@@ -0,0 +1,9 @@
+<ng-container [formGroup]="parentGroup">
+  <label class="usa-input" id="quantity-label">{{ label }}</label>
+  <span id="quantity-hint-text" class="help-text usa-form-hint"><span *ngIf="hintText">{{hintText}}</span></span>
+  <input id="quantity" class="usa-input-tiny" type="number" min='1' max="{{length}}" formControlName="quantity" aria-required="true"
```

Comment:
{{ length }} seems bit confusing as the maximum number of trees? Its is an array of valid options? Any way to rename this?

Reply:
@lauraGgit this is not the max number of trees, it's the number of characters. We can remove it altogether, and replace with '1' since we don't have any forests with more than 9 trees allowed. Let us know if you would like us to make that refactor